101-servicebus-namespace/azuredeploy.parameters.json

```diff
@@ -1,9 +1,12 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "serviceBusNamespaceName": {
-            "value": "GEN-UNIQUE"
-        }
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceBusNamespaceName": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusSku": {
```

Comment:
remove this and just use the defaultValue

Reply:
resolved